app/src/main/java/org/glucosio/android/activity/AddGlucoseActivity.java

```diff
@@ -106,9 +110,9 @@ public void onNothingChosen(View labelledSpinner, AdapterView<?> adapterView) {
             }
         });
 
-        TextView unitM = (TextView) findViewById(R.id.glucose_add_unit_measurement);
+        TextView unitM = findViewById(R.id.glucose_add_unit_measurement);
 
-        if (presenter.getUnitMeasuerement().equals("mg/dL")) {
+        if (presenter.getUnitMeasuerement().equals(DomainConstants.MG_D_L)) {
```

Comment:
Typo: `Measuerement` should be `Measurement`

Reply:
I agree with @WBertan

Reply:
Fixed!